### PR TITLE
Update php-laravel devfile

### PR DESCRIPTION
**Pull request by dkwon17**
### What does this PR do?:
_Summarize the changes. Are any stacks or samples added or updated?_
1. Created new version folder 2.0.1
2. Updated schema version to 2.2.2
3. Updated endpoint protocol to https
4. Set the default version to 2.0.1 in stack.yaml

### Which issue(s) this PR fixes:
_Link to github issue(s)_

### PR acceptance criteria:

- [ ] Contributing guide
_Have you read the [devfile registry contributing guide](https://github.com/devfile/registry/blob/main/CONTRIBUTING.md) and followed its instructions?_
- [ ] Test automation
_Does this repository's tests pass with your changes?_
- [ ] Documentation
_Does any documentation need to be updated with your changes?_
- [ ] Check Tools Provider
_Have you tested the changes with existing tools, i.e. Odo, Che, Console? (See [devfile registry contributing guide](https://github.com/devfile/registry/blob/main/CONTRIBUTING.md) on how to test changes)_

### How to test changes / Special notes to the reviewer:

### CODE REVIEW

File: stacks/php-laravel/2.0.1/devfile.yaml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  - name: php-laravel-starter
+    git:
+      checkoutFrom:
+        revision: main
+      remotes:
+        origin: https://github.com/devfile-samples/devfile-stack-php-laravel.git
+components:
+  - container:
+      endpoints:
+        - name: http-laravel
```
Comment (thepetk) [inline stacks/php-laravel/2.0.1/devfile.yaml:28]:
```suggestion
        - name: https-laravel
```

**Review by thepetk — CHANGES_REQUESTED** [review]
Suggested a minor change

**Comment by dkwon17** [comment]
Thank you, I fixed the changes, but I had to force push to my branch to remove the unsigned commit

**Comment by thepetk** [comment]
> Thank you, I fixed the changes, but I had to force push to my branch to remove the unsigned commit

Thanks! The only thing remains from our side is to address the failing check issue https://github.com/devfile/api/issues/1494

/hold

**Review by thepetk — APPROVED** [review]
/lgtm

**Comment by thepetk** [comment]
@dkwon17 could you please rebase with the latest main. I think the issue with the odov3 check will be fixed

**Comment by thepetk** [comment]
Tested locally and it looks good.

**Pull request merged**